### PR TITLE
DA_Week4_복습과제_오연재

### PR DESCRIPTION
**Pull request by jennyohyj**

### CODE REVIEW

**Comment by lej3034** [comment]
안녕하세요 연재님! 4주차 복습과제 피드백 드리겠습니다 :)

wine_data.type.astype("category").cat.codes 를
pd.get_dummies(X_wine['type'])로 해도 같은 결과를 얻을 수 있고

XGBoost를 사용해서 train 세트를 fit하고 test 세트로 예측 정확도 및 F1스코어 출력 문제에서는
w_pred_proba = xgb_wrapper.predict_proba(X_test)[:, 1]와 get_clf_eval(y_test , w_preds, w_pred_proba)를  삭제하고
print('모델의 정확도:', accuracy_score(y_test, y_preds))
print('모델의 F1 스코어:', f1_score(y_test, y_preds, average='weighted')) 를 넣어주고

LightGBM과 GridSearchCV로 최적 하이퍼 파라미터 조합과 예측 정확도 구하는 문제에서는
GridSearchCV(lgbm_wrapper,parameters,scoring='accuracy')를
GridSearchCV(lgbm_wrapper, param_grid=parameters, cv=2, verbose=1)로 바꿔주고 

CatBoost 모델 예측 성능 평가 문제에서는 
accuracy = accuracy_score(y_test , catb_preds)
accuracy
를 넣어주면 잘 실행되는 것 같아요

수고 많으셨습니다 !!
